lib/tasks/search-term-report.rake

```diff
@@ -0,0 +1,11 @@
+namespace :report do
+  desc "Run a publication delay report of the last week"
+  task publication_delay_report: :environment do
+    PublicationDelayReport.call($stdout)
+  end
+
+  desc "Find all references to search term"
```

Comment:
We can move this task into the `report.rake` file.

Reply:
Could do, but we only really want to run one or the other (we don't want the gsi report every time)

Reply:
What you can do is add

```
desc "Find all references to search term"
task :find_specific_term_references, %i[term] => [:environment] do |_, args|
  FindSpecificTerm.call(args[:term])
end
```

into the file `report.rake` under the `namespace :report` and that allows people to run only the task `report:find_specific_term_references` without having to run the GSI task as well. And it means we can get rid of this file.

Reply:
Done and done.